push-notifications/android-migrate-fcm.md

```diff
@@ -0,0 +1,244 @@
+---
+title: Migrate to FCM from GCM
+---
+
+[[toc]]
+
+As of April 10, 2018, [Google has deprecated GCM][google-migrate-fcm-faq]. For apps that are using GCM in Skygear for Android push notifications, you can follow the below steps to migrate from GCM to FCM.
+
+1. Update Skygear server to v1.7.0+
+1. Migrate from GCM setup to FCM in android client
+
+## Update Skygear server
+
+After updating Skygear server to v1.7.0+, Skygear server will use the configured server key with FCM API automatically. There's no need to reconfigure.
```

Comment:
Should we also tell users that they can switch to using the service account if they want to?